cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -52,6 +53,8 @@ type StaticAutoscaler struct {
 	// AutoscalingContext consists of validated settings and options for this autoscaler
 	*context.AutoscalingContext
 	kube_util.ListerRegistry
+	// ClusterState for maintaining the state of cluster nodes.
+	ClusterStateRegistry    *clusterstate.ClusterStateRegistry
```

Comment:
Why does it need to be public?

Reply:
It doesn't, it just was, similar to the two above. Changed that.